src/App.js

```diff
@@ -4,7 +4,7 @@ import GameShelf from './GameShelf.js';
 import Footer from './Footer.js';
 import './App.css';
 
-class App extends Component {
+export default class App extends Component {
```

Comment:
Adding 'export default' to the beginning of our class declarations is a great way to clean up code and remove an unnecessary line, good thinking